Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM swaggerapi/swagger-ui:v5.9.1 AS swagger-ui
 FROM python:3.10-slim
 
+RUN useradd -m service
```

Comment:
This maps to an arbitrary user id that may or may not exist on the Docker host, right? So if you want asr to, for example, cache models in mounted volumes (mounted to host folders), those folders on the host would need to be owned by the same user id and have proper permissions.